core/build/webpack.base.config.js

```diff
@@ -40,7 +44,19 @@ const postcssConfig =  {
 module.exports = {
   plugins: [
     new CaseSensitivePathsPlugin(),
-    new VueLoaderPlugin()
+    new VueLoaderPlugin(),
+    // generate output HTML
```

Comment:
maybe this files should be provided in config or exported from some place that develoeprs can reach (not from core)?